admin/spec/service/group.service.spec.js

```diff
@@ -57,6 +70,18 @@ describe('group.service', () => {
       expect(pupils).toEqual(pupilsMock)
       done()
     })
+
+    it('should return an error if schoolId is missing', async (done) => {
+      const schoolId = null
+      const groupIdToExclude = 1
+      try {
+        await groupService.getPupils(schoolId, groupIdToExclude)
+        done()
```

Comment:
`done()` is common to both code paths and could be moved after the try/catch block.  You also need a `fail()` check to ensure that the test fails if an error is not thrown.

Also, you don't strictly need the `done` call as you are not using callbacks, `await` makes the code execute in sequence.